app/views/assignments/show.html.erb

```diff
@@ -32,206 +32,201 @@
       <%= render partial: 'read', locals:{ peer_review: false } %>
     </div>
   </div>
+  <% if @assignment.group_max > 1 %>
+    <div class='pane block'>
+      <h2><%= t('groups.student_interface.group_information') %></h2>
+      <div class='block-content'>
+        <% # If student has no group, inform them
+           if @grouping.nil? %>
+          <p class='notice'>
+            <% if @assignment.submission_rule.type == GracePeriodSubmissionRule && @assignment.group_max > 1%>
+              <%= t('groups.student_interface.no_group_yet') %>
+            <% end %>
+            <% if !@assignment.scanned_exam && @assignment.past_collection_date?(@current_role.section) %>
+              <%= t('assignments.due_date.final_due_date_passed') %>
+            <% end %>
+          </p>
 
-  <div class='pane block'>
-    <h2><%= t('groups.student_interface.group_information') %></h2>
-
-    <div class='block-content'>
-      <% # If student has no group, inform them
-         if @grouping.nil? %>
-        <p class='notice'>
-          <%= t('groups.student_interface.no_group_yet') %>
-          <% if !@assignment.scanned_exam && @assignment.past_collection_date?(@current_role.section) %>
-            <%= t('assignments.due_date.final_due_date_passed') %>
+          <% if @assignment.is_timed && @assignment.group_max == 1 %>
+            <p><%= t('groups.student_interface.create_individual_folder_on_start') %></p>
```

Comment:
This block will no longer ever appear (because of the `group_max > 1` check you added). You can just delete this if block and string, it doesn't add too much. But make sure to keep the `elsif` block below.